SIL.ReleaseTasks/SetReleaseNotesProperty.cs

```diff
@@ -71,6 +73,10 @@ public override bool Execute()
 
 		private string ConvertLatestChangelog(int level, int skipUntilLevel = -1)
 		{
+			string urlRegex = @"(\[([^]]+)\]: (http|https|ftp|)://[0-9a-zA-Z]([-.\w]*[0-9a-zA-Z])*(:(0-9)*)*(/?)" +
+					@"([-a-zA-Z0-9.?,'/\+&%$#_]*)?([-a-zA-Z0-9?,'/+&%$#_]+))+";
+			_urlRegex = new Regex(urlRegex);
```

Comment:
If we create the regex here we don't need a member variable. However, creating a regex is fairly expensive, so we should use the member variable and create the regex in `Execute()`.